tests/formattedcode/data/json/tree/expected.json

```diff
@@ -329,7 +345,10 @@
       "is_source": true,
       "is_script": false,
       "licenses": [],
+      "unknown_licenses": [],
       "license_expressions": [],
+      "unknown_license_expressions": [], "license_expressions": [],
+      "unknown_license_expressions": [],
```

Comment:
This is not right: you cannot have twice the same keys? This is 
```
      "unknown_licenses": [],
      "license_expressions": [],
      "unknown_license_expressions": [], "license_expressions": [],
      "unknown_license_expressions": [],

```

and the order should be:
```
      "licenses": [],
      "license_expressions": [],
      "unknown_licenses": [],
      "unknown_license_expressions": [],
```